oletools/olemeta.py

```diff
@@ -60,8 +60,8 @@
 #=== IMPORTS =================================================================
 
 import sys, codecs
-import thirdparty.olefile as olefile
-from thirdparty.tablestream import tablestream
+import .thirdparty import olefile
```

Comment:
I mean there is just a typo: the 1st "import" should be "from" instead. :-)
i.e. `from .thirdparty import olefile`